examples/kotlinExample/build.gradle

```diff
@@ -29,24 +41,10 @@ android {
     }
 }
 
-tasks.preBuild {
-    dependsOn ":realm:androidJar"
-}
-
 dependencies {
-    compile "org.jetbrains.kotlin:kotlin-stdlib:$kotlin_version"
+    compile "org.jetbrains.kotlin:kotlin-stdlib:${kotlin_version}"
     compile 'org.jetbrains.anko:anko:0.6.2-19s'
-    compile files("../../realm/build/libs/realm-${version}.jar")
-
-    kapt files("../../realm/build/libs/realm-${version}.jar")
+    compile "io.realm:realm-android:${version}@aar"
```

Comment:
I prefer being explicit here just in case we end up deploying the jar with the same archive name by mistake